src/config.ts

```diff
@@ -27,7 +27,7 @@ const config: { [key: string]: any } = {
   // In production postgraphile is started with -q and -i /postgraphile/...
   nodeModulesFolder:
     process.env.NODE_ENV === 'production' ? '../../node_modules' : '../node_modules',
-  jwtSecret: process.env.JWT_SECRET,
+  jwtSecret: process.env.JWT_SECRET || 'devSecret',
```

Comment:
Maybe needs to be removed?

Reply:
No, I put this in to stop the server crashing if no secret was supplied in .env. This default is useful for development, cos it means all our existing JWTs keep working (for example in test suites, and Postman collections). But we should always be adding a secure randomly generated secret key whenever we update and re-start the server, so there's no problem leaving this "default" one out in plain view as it should never be used on a live server.

Reply:
Cool. Thanks for explaning this